script/plugin_tools.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -ev
+
+if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
+  export FLUTTER_CHANGED_PACKAGES=""
+else
+  export FLUTTER_CHANGED_PACKAGES=`git diff --name-only $TRAVIS_COMMIT_RANGE | grep -o "packages/[^/]*" | sed -e "s/packages\///g" | sort | uniq | paste -s -d, -`
```

Comment:
Maybe I am missing something or don't understand how this is supposed to work? I don't see how this would run on all packages if you change files inside *and* outside `packages` in the same PR.